src/backend/distributed/citus--7.5-4--7.5-5.sql

```diff
@@ -0,0 +1,34 @@
+/* citus--7.5-4--7.5-5 */
+CREATE FUNCTION citus.citus_executor_name(executor_type int)
```

Comment:
We may need to be careful about this function being in the citus dchema, since regular users don't have access to it.

Reply:
pg_catalog looked more restrictive. What do you suggest ?

Reply:
pg_catalog seems fine, all users have usage access to pg_catalog

Reply:
updated